docs/index.md

```diff
@@ -29,6 +29,11 @@ The following 3 license types (`jq .type`) do **NOT** support APIs:
 - JCR Edition
 - OSS
 
+## Versions
+
+We maintain two major versions of Terraform Provider - 6.x and 7.x. Version 6.x is compatible with the Artifactory versions 7.49.x and below,
```

Comment:
Do we want to use `~>` sigil to highlight this?

Reply:
We can, why not. Added.